package.json

```diff
@@ -3,29 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "start": "node index.js",
-    "serve": "vue-cli-service serve",
+    "start": "nodemon index.js",
+    "serve": "PORT=8081 vue-cli-service serve",
+    "dev-start": "concurrently \"npm run start\" \"npm run serve\"",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "test": "jest --updateSnapshot --projects src app --no-cache --detectOpenHandles --json --outputFile=./coverage/coverage.json"
-  },
-  "jest": {
-    "projects": [
-      "src/jest.config.js",
-      "app/jest.config.js"
-    ]
+    "test:node": "jest --config app/jest.config.js app --no-cache --json --outputFile=./coverage/coverage.json",
+    "test:vue": "jest --updateSnapshot --projects src --no-cache",
+    "test": "npm run test:node && npm run test:vue"
   },
   "dependencies": {
     "@gofynd/fdk-client-javascript": "^1.3.5",
     "@gofynd/nitrozen-vue": "0.0.33",
     "axios": "^0.27.2",
+    "axios-mock-adapter": "^1.21.2",
     "body-parser": "^1.20.0",
     "convict": "^6.2.3",
```

Comment:
@jinalviranii please remove dependency of convict

Reply:
@vivek-gofynd Why we want to remove this? We discussed to not remove if you remember